components/ILIAS/LegalDocuments/classes/Conductor.php

```diff
@@ -89,14 +74,19 @@ public function loginPageHTML(string $id): string
     public function logoutText(): string
     {
         try {
```

Comment:
The try catch block is only for `...->query->retrieve(...)` not the whole method.